service/models.py

```diff
@@ -73,10 +72,14 @@ def update(self, new_data):
         Updates a Inventory to the database
         """
         self.name = new_data.name or self.name
-        self.quantity = new_data.quantity or self.quantity
+        # if hasattr(new_data, 'quantity'):
```

Comment:
if this is not required, you can delete it in later commits

Reply:
fixed it